lib/improver/nowcasting/optical_flow.py

```diff
@@ -119,6 +119,11 @@ def __init__(self, vel_x, vel_y):
         self.x_coord = vel_x.coord(axis="x")
         self.y_coord = vel_x.coord(axis="y")
 
+    def __repr__(self):
+        """Represent the plugin instance as a string."""
+        result = ('<AdvectField>')
+        return result
```

Comment:
It might be useful to print out the time coordinate from the vel_x cube to show the user what time the initialised object is valid for.

Reply:
I haven't actually required an input time coordinate on the velocity cube.  This could be updated but it's a little bit of work, including changing the plugin functionality to require a time coordinate.  Do we want to do this?